src/utils/index.ts

```diff
@@ -214,7 +214,21 @@ function ISOTimeToRippleTime(iso8601: string): number {
   return unixToRippleTimestamp(Date.parse(iso8601))
 }
 
+/**
+ * It returns a function that prepends params to the given func.
+ * A sugar function for JavaScript .bind().
+ *
+ * @param func - A function to prepend params.
+ * @param params - Parameters to prepend to a function.
+ * @returns A function bound with params.
+ */
+function partial(func: Function, ...params: any[]): Function {
```

Comment:
Given that this is a helper method for the client, I'd recomment putting this in `src/client` so it doesn't need to be exported.

Reply:
I put it here because I see we use Lodash's version of `partial` in other files too. I'm planning on creating another PR to replace all Lodash partial with this one.

Reply:
Everywhere we're using it will be scrapped, so don't worry about that. You can even make the method tailored to the client.